app/config/providers.php

```diff
@@ -61,6 +61,16 @@
         'beta' => false,
         'mock' => false
     ],
+    'dailymotion' => [
+        'name' => 'Dailymotion',
+        'developers' => 'https://developers.dailymotion.com/api/',
+        'icon' => 'icon-dailymotion',
```

Comment:
Can we please increase the padding of the PNG? It is getting into rounded borders

<img width="102" alt="CleanShot 2022-06-13 at 10 04 29@2x" src="https://user-images.githubusercontent.com/19310830/173308120-5386fc53-1feb-44c8-980e-c0224ed2d46b.png">

Reply:
Sorry to be picky, but can we please make the padding a bit bigger? I would say it is fine to even make it small like Facebook or GitHub.

Also, our design team noticed white borders on left&right of the circle on a dark theme. Can we look at that? Or is that just the way browser rendered it on your resolution?

Reply:
I guess these white borders are due to the export issues. I am just asking my friend for help. Will ping you again in few hours